apisix/plugins/jwe-decrypt.lua

```diff
@@ -0,0 +1,234 @@
+--
+-- Licensed to the Apache Software Foundation (ASF) under one or more
+-- contributor license agreements.  See the NOTICE file distributed with
+-- this work for additional information regarding copyright ownership.
+-- The ASF licenses this file to You under the Apache License, Version 2.0
+-- (the "License"); you may not use this file except in compliance with
+-- the License.  You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+local core            = require("apisix.core")
+local consumer_mod    = require("apisix.consumer")
+local base64          = require("ngx.base64")
+local ngx             = ngx
+local sub_str         = string.sub
+local plugin_name     = "jwe-decrypt"
+local cipher          = require("resty.openssl.cipher").new("aes-256-gcm")
```

Comment:
Why not use https://github.com/openresty/lua-resty-string?

Reply:
@fishioon have you noticed this comment?

Reply:
@shreemaan-abhishek

Reply:
Is there any issue in using the current method?

Reply:
if we use `lua-resty-string`, we don't need to import any new library